pkg/cmd/codespace/list.go

```diff
@@ -73,7 +73,11 @@ func (a *App) List(ctx context.Context, limit int, exporter cmdutil.Exporter) er
 			stateColor = cs.Green
 		}
 
-		tp.AddField(c.Name, nil, cs.Yellow)
+		nameField := c.Name
+		if c.DisplayName != "" {
+			nameField = fmt.Sprintf("%s (%s)", nameField, c.DisplayName)
```

Comment:
Actually, no. We can't have this, because it breaks scripts.

The unwritten contract is that the first column is the unique identifier for the codespace. Primarily, this value needs to be usable for the `-c` flag:
```
gh codespace logs -c mislav-github-github-vrgg3pg64
```
When we insert the display name into the value, it doesn't work:
```
gh codespace logs -c 'mislav-github-github-vrgg3pg64 (jubilant space succotash)'
```
That's why we can't modify the first column of the `list` command. It's better that the list command stays exactly as it was before until we figure out how to shoehorn the display name in.

Reply:
Okay, the displayname is only available when using `list --json`

```bash
@veverkap ➜ /workspaces/cli (add_display_name) $ bin/gh cs list --json displayName
[
  {
    "displayName": "refactored chainsaw"
  }
]
@veverkap ➜ /workspaces/cli (add_display_name) $ bin/gh cs list
NAME                         REPOSITORY  BRANCH            STATE      CREATED AT
veverkap-cli-cli-qjv972x6xj  cli/cli     add_display_name  Available  3d
```